ftw/testing/freezer.py

```diff
@@ -43,6 +56,7 @@ def freezed_now(klass, tz=None):
                 return self.new_now
 
         curse(datetime, 'now', freezed_now)
+        curse(datetime, 'utcnow', freezed_now)
```

Comment:
It is actually possible to freeze to a datetime with a tzinfo (compareable with the server timezone).
`freezed_now` will then just remove the tzinfo and return local time when no tz-parameter is used, see the implementation above.

I think it would therefore be consistent to convert to utc for utcnow in the situation when the frozen time has a timezone.

This can actually be done by reusing the `freezed_now`:
e.g.:
```python
         def freezed_utcnow(klass):
             return freezed_now(pytz.UTC).replace(tzinfo=None)
```

What do you think, does this make sense?

Reply:
Hmm, I think not: `freezed_now` is a classmethod with one argument `tz`, matching the implementation of `datetime.datetime.now`.
`datetime.datetime.utcnow` on the other hand is a classmethod without arguments.
=> https://docs.python.org/2/library/datetime.html#datetime.datetime.now

`curse` applies the patch (a bit like `setattr`), so there should happen much magic here.

Or did I missunderstand you?

Reply:
You did not. This is simpler than I originally thought. Thank you. Will amend.